output_vt100.go

```diff
@@ -270,6 +270,11 @@ func (w *VT100Writer) SetDisplayAttributes(fg, bg Color, attrs ...DisplayAttribu
 	return
 }
 
+// UseStderr to set stderr as output
+func (w *VT100Writer) UseStderr() {
+	// Do nothing
```

Comment:
I'm not understanding VT100 at all. Could you help?

Reply:
VT100 escape sequences can just decorate texts which are printed and control the views (ex: change cursor position, erase screen and others listed on https://vt100.net/docs/vt100-ug/chapter3.html) in Terminal emulators. So it doesn't relate to which standard streams we use. So it's ok you only changes `output_posix.go` and `output_windows.go` and `option.go`.

Reply:
Thank you. Let me check.